docs/packages/pkg/rocksdb.rst

```diff
@@ -0,0 +1,23 @@
+.. spelling::
+
+    rocksdb
+    Paweł
+    Bylica
+
+.. index:: database ; rocksdb
+
+.. _pkg.rocksdb:
+
+rocksdb
+=======
+
+-  `Webpage <http://rocksdb.org>`__
+-  `Source code <https://github.com/facebook/rocksdb>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/rocksdb/CMakeLists.txt>`__
+-  Added by Paweł Bylica `chfast <https://github.com/chfast>`__ (`pr-1231 <https://github.com/ruslo/hunter/pull/1231>`__)
```

Comment:
That's confusing. Either substitute it with yours or add `Developer-1 and Developer-2` or `started by Developer-1 finished by Developer-2` or just only leave the link to pull request.